src/Catalyst.Cli.Tests/UnitTests/Commands/Response/GetInfoResponseTests.cs

```diff
@@ -42,7 +42,7 @@ public void GetInfoResponse_Can_Get_Output()
             //Arrange
             var getInfoResponse = new GetInfoResponse {Query = "Test"};
             var commandContext = TestCommandHelpers.GenerateCliResponseCommandContext(_testScheduler);
-            var getInfoCommand = new GetInfoCommand(commandContext, Substitute.For<ILogger>());
+            new GetInfoCommand(commandContext, Substitute.For<ILogger>());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove this useless object instantiation of class 'GetInfoCommand' or use it.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)